### PR TITLE
Remove dependency of Bert from Squeezebert tokenizer

### PR DESCRIPTION
**Pull request by rchan26**
Hi @sgugger,

Fixes #19303, the BertTokenizer dependency has been removed from `SqueezeBertTokenizer` and the BertTokenizerFast dependency has been removed from `SqueezeBertTokenizerFast`.

I ran `pytest tests/models/squeezebert/test_tokenization_squeezebert.py`, which passed.

Thanks for reviewing this! :)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

### CODE REVIEW

**Comment by rchan26** [comment]
I see that my code currently fails the style and code consistency checks. When I run `make style`, it seems to change a lot of files and ones that I did not touch. Is this normal?

I'm also trying to run `make repo-consistency` but keep getting
```
python utils/check_copies.py --fix_and_overwrite
make: python: No such file or directory
make: *** [fix-copies] Error 1
```
which is strange as I am running this from the root directory...

File: src/transformers/models/squeezebert/tokenization_squeezebert.py [hunk 1]
```diff
@@ -48,17 +53,476 @@
...
+def whitespace_tokenize(text):
+    """Runs basic whitespace cleaning and splitting on a piece of text."""
+    text = text.strip()
+    if not text:
+        return []
+    tokens = text.split()
+    return tokens
+
+
+class SqueezeBertTokenizer(PreTrainedTokenizer):
```
Comment (sgugger) [inline src/transformers/models/squeezebert/tokenization_squeezebert.py:78]: I think you can add a copied from statement here
```suggestion
# Copied from transformers.models.bert.tokenization_bert.BertTokenizer with Bert->SqueezeBert,BERT->SqueezeBERT
class SqueezeBertTokenizer(PreTrainedTokenizer):
```

File: src/transformers/models/squeezebert/tokenization_squeezebert.py [hunk 2]
```diff
@@ -48,17 +53,476 @@
...
+            Whether or not to strip all accents. If this option is not specified, then it will be determined by the
+            value for `lowercase` (as in the original BERT).
     """
 
     vocab_files_names = VOCAB_FILES_NAMES
     pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
     max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
     pretrained_init_configuration = PRETRAINED_INIT_CONFIGURATION
+
+    # Copied from transformers.models.bert.tokenization_bert.BertTokenizer.__init__
```
Comment (sgugger) [inline src/transformers/models/squeezebert/tokenization_squeezebert.py:126]: With the copied from on the whole class, you won't need to apply it on each method.

File: src/transformers/models/squeezebert/tokenization_squeezebert_fast.py [hunk 3]
```diff
@@ -60,18 +65,153 @@
 }
 
 
-class SqueezeBertTokenizerFast(BertTokenizerFast):
+class SqueezeBertTokenizerFast(PreTrainedTokenizerFast):
```
Comment (sgugger) [inline src/transformers/models/squeezebert/tokenization_squeezebert_fast.py:68]: Same here!

**Review by sgugger — COMMENTED** [review]
Thanks a lot for your PR! I left a couple of comments on the copied from statement.

For the error with `make repo-consistency`, are you sure you are running this from the root folder?

For `make style`, make sure you have the same version of `black` as we use by running `pip install -e .[quality]`

File: src/transformers/models/squeezebert/tokenization_squeezebert_fast.py (hunk 3 above)
Comment (rchan26) [reply src/transformers/models/squeezebert/tokenization_squeezebert_fast.py:68]: Do you mean to write something like `# Copied from transformers.models.bert.tokenization_bert.BertTokenizerFast with Bert->SqueezeBert,BERT->SqueezeBERT` above this?

Comment (sgugger) [reply src/transformers/models/squeezebert/tokenization_squeezebert_fast.py:68]: Yes!

**Comment by rchan26** [comment]
Hi @sgugger, many thanks for the quick replies! I have made the changes you mentioned above, and regarding `make repo-consistency` and `make-style`, it seems like `pip install -e ."[quality]"` did the trick!

**Review by sgugger — APPROVED** [review]
Thanks again for all your work on this!

**Pull request merged**